static/js/publisher/builds/index.js

```diff
@@ -1,8 +1,117 @@
 import React from "react";
 import ReactDOM from "react-dom";
 
+import distanceInWords from "date-fns/distance_in_words_strict";
+
 import MainTable from "@canonical/react-components/dist/components/MainTable";
 
+const NEVER_BUILT = "never_built";
```

Comment:
We could maybe move these out in future, as we do for releases - but for now, no need!

Reply:
Yes, this should be split into smaller pieces sooner than later